### DISCUSSION TITLE
This project is amazing

### DISCUSSION THREAD

**Discussion by lianjinke** [jart/cosmopolitan#968] [General]
Ah, are C++ and Rust supported, by the way?

**Comment by jart** [comment]
Thank you! C++ is supported. See the `cosmoc++` compiler that comes with cosmocc. Rust is blocked on https://github.com/rust-lang/rfcs/pull/3305#issuecomment-1238524298

**Reply by lianjinke** [reply]
I see it. Thanks buddy! Seems they don't plan to do something?
